framework/doc/content/source/postprocessors/TimePostprocessor.md

```diff
@@ -0,0 +1,9 @@
+# TimePostprocessor
+
```

Comment:
Could we explain this object using one sentence?

Reply:
I don't like the redundancy that that brings. See [this postprocessor's](https://mooseframework.inl.gov/docs/PRs/15614/site/source/postprocessors/TimePostprocessor.html) doc page, vs. the one for [`TimestepSize`](https://mooseframework.inl.gov/docs/PRs/15614/site/source/postprocessors/TimestepSize.html) which has an explicit description in the `.md` file. If I didn't have the `!syntax description /Postprocessors/TimePostprocessor` I would absolutely agree with you.